### PR TITLE
Weather App

### PR DESCRIPTION
**Pull request by marcode95**
## Weather App
-app uses the openweather api
-users can input a location of their choice to check on the current temperature there
-users can switch between temperature units

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @marcode95  👋

**You've done pretty well here, clean code and nice idea showing a default city's weather info on initial page load. 👌**

However, let's make a couple of changes.

- [ ] In the readme, please include exact steps for a user to install packages and run the project locally.

- [ ] OPTIONAL - Consider adding a loading animation while retrieving weather info.

- Please refer to other comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  try {
+    const response = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=3d01fe516dd3a560412d2f557ad5d8f5`, { mode: 'cors' });
+    const responseData = await response.json();
+    // eslint-disable-next-line max-len
+    const newItem = weatherData(responseData.name, responseData.main.temp, responseData.weather[0].main);
+    append(newItem);
+  } catch (error) {
+    errorWindow.classList.remove('display-none');
+    errorWindow.classList.add('display-block');
+    setTimeout(() => {
```
Comment (mikenath223) [inline src/index.js]: - [x] Kindly change this to es6 arrow function syntax.

**Comment by marcode95** [comment]
I turned the async function into a proper es6 function. I will not change the readme, since it already contains a link to a detailed explanation on how to clone repos.

**Review by arslanbisharat — APPROVED** [review]
# Approved :clap:

Hello Developer, :wave:

Thank You for submitting this Pull Request. I am thrilled to inform you that your project is approved now. :clap:

You have done a great job. I like your project. 

**To highlight** 

- [ ] You have a descriptive Readme.
- [ ] Your Pull Request is nice. Good Job!
- [ ] You have deployed your website. Nice!!!
- [ ] You are using linters and no error is present. I am happy to see it. 
- [ ] Your project has met the minimum technical requirements too. 

In short, I am very excited to see this project and proud of you. 

I am wishing you very good luck with the next project. :smile:

Once again, Congratulations!:joy:

**Best,
Arslan Bisharat(PT-TSE)
Slack: @arslanbisharat**

**Pull request merged**